simbad/rotsearch/phaser_search.py

```diff
@@ -336,3 +341,19 @@ def rot_succeeded_log(self, log):
                             self.solution = True
                             return True
         return False
+
+    def percentage_complete(self):
+        total_log_files = 0
+        log_files = glob.glob(os.path.join(self.script_log_dir, '*.log'))
+        for log in log_files:
+            with open(log, 'r') as f:
+                for line in f:
+                    if line.startswith("EXIT STATUS: SUCCESS"):
+                        total_log_files += 1
+        total_sh_files = len(glob.glob(os.path.join(self.script_log_dir, '*.sh')))
+        percentage_complete = (total_log_files / total_sh_files) * 100
+        if percentage_complete - self.progress >= 5:
+            logger.info("Percentage complete: {:.1f}%".format(percentage_complete))
+            self.progress = percentage_complete
+        return
```

Comment:
Same as in amore_search.py

Reply:
Actually you could refactor this function somewhere and import it rather than have it copy pasted twice in the code

Reply:
I’m planning on making a parent class for phaser_search and amore_search at some point. They share a lot of functions. That’s a job for another day though.

Reply:
Sounds reasonable, these two classes share a lot of similarities.